packages/core/hooks/useOpenWithMenuItems/useOpenInCfe.ts

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import { RemoteFileUploadServerConnection } from "../useRemoteFileUpload";
+import { uniqueId } from "lodash";
+import Annotation from "../../entity/Annotation";
+import FileSelection from "../../entity/FileSelection";
+import { FileService } from "../../services";
+import { interaction } from "../../state";
+import { useDispatch } from "react-redux";
```

Comment:
nit: (We need to document this, sorry) the general pattern for imports in BFF is 
- external -- alphabetized by module name
- space
- internal -- by relative path distance and then alphabetized, e.g., `./`, then `../a`, then `../b`, then `../../a` etc
- space
- style.css files 
- IF it's a .test.ts or .test.tsx file, the file that's being tested